chart/kubeapps/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kubeapps
-version: 0.1.0
+version: 0.1.2
```

Comment:
This PR depends on https://github.com/kubeapps/kubeapps/pull/484 that's why it bumps twice.